### PR TITLE
Translation cleanup

### PR DESCRIPTION
**Pull request by dacook**
#### What? Why?

Translation code cleanup. 
I was going to add a new side menu item for Enterprises, but noticed the existing translations don't follow the [guidelines](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-%28i18n%29#development) at all. In order to add new code that meets the guidelines (and learn how it all works), I had a go at updating this first.

This removes some global keys, and creates some new more specific ones.

#### What should we test?

The following pages are changed to use new, specifically scoped keys. 

* [New Enterprise](http://localhost:3000/admin/enterprises/new)
  **All labels**
* [Enterprise Edit](http://localhost:3000/admin/enterprises/mary-s-online-shop/edit)
   The text for each **side menu item** and **form legend title** (primary details, address, etc)
* [Enterprise Group Edit](http://localhost:3000/admin/enterprise_groups/producer-group/edit)
   The text for each **side menu item** and **form legend title** (primary details, address, etc)

The following global keys are now unused and have now been deleted:

```yaml
adrdress: "Address"

enterprise_permissions: "Enterprise Permissions"
business_details: "Business Details"
properties: "Properties"
inventory_settings: "Inventory Settings"
shop_preferences: "Shop Preferences"

admin_enterprise_groups_contact: "Contact"
admin_enterprise_groups_web: "Web Resources"
```
(I looked as hard as I could to check they're not used, but could be wrong)

#### Release notes

Changelog Category: Technical changes

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -2826,7 +2826,6 @@ See the %{link} to find out more about %{sitename}'s features and to start using
   contact: "Contact"
   web: "Web"
   primary_details: "Primary Details"
-  adrdress: "Address"
```
Comment (mkllnk) [inline config/locales/en.yml:2829]: It actually contains a typo. Very unlikely that it's used if you can't find the string.

**Review by mkllnk — APPROVED** [review]
Nice cleanup, thank you!

**Review by jibees — APPROVED** [review]
Nice cleanup! Thanks a lot! 👌

**Comment by dacook** [comment]
Rebased with the new ["Update branch" button](https://github.blog/changelog/2022-02-03-more-ways-to-keep-your-pull-request-branch-up-to-date/) :D

**Comment by drummer83** [comment]
Hi @dacook,
Thanks for this cleanup! This was bugging me some time ago as well... :-)

I have tested the pages you mentioned above on our staging AU server (it should use en.yml):
- Enterprise edit
- Enterprise group edit
- New enterprise

I have also looked for places where the deleted strings might have been needed - everything I could think of is still ok.

So far I would say we are ready to go here! 🎉 

@filipefurtad0: Is this something we should test again after releasing?

**Pull request merged**